src/utils/getUserDisplayName.ts

```diff
@@ -0,0 +1,19 @@
+import { User } from 'oidc-client-ts';
+
+import skipFalsyType from './skipFalsyType';
+
+const getUserDisplayName = ({
```

Comment:
Here is another one where all changes related to the introduction of this util function could be in their own refactoring commit.